pkg/antctl/raw/packetcapture/command_test.go

```diff
@@ -0,0 +1,280 @@
+// Copyright 2025 Antrea Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package packetcapture
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	"k8s.io/client-go/kubernetes"
+	k8sfake "k8s.io/client-go/kubernetes/fake"
+	"k8s.io/client-go/rest"
+	k8stesting "k8s.io/client-go/testing"
+	"k8s.io/utils/ptr"
+
+	"antrea.io/antrea/pkg/antctl/raw"
+	"antrea.io/antrea/pkg/apis/crd/v1alpha1"
+	antrea "antrea.io/antrea/pkg/client/clientset/versioned"
+	antreafakeclient "antrea.io/antrea/pkg/client/clientset/versioned/fake"
+)
+
+const (
+	srcPod            = "default/pod-1"
+	dstPod            = "pod-2"
+	ipv4              = "192.168.10.10"
+	testNum     int32 = 10
+	testTimeout       = 3 * time.Second
```

Comment:
`3s` is usually a bit too long when it comes to unit tests IMO. Could you use a fake clock for this?
Unfortunately, `wait.PollUntilContextTimeout` doesn't support a clock as a parameter, so we have to use the underlying functions directly:

```go
	b := wait.Backoff{
		Duration: 1 * time.Second,
	}
	ctx, cancel := context.WithTimeout(ctx, timeout+5*time.Second)
	defer cancel()
	err := b.DelayWithReset(c, 0).Until(ctx, true, false, func(ctx context.Context) (bool, error) {
		fmt.Println("STEP")
		return false, nil
	})
	if wait.Interrupted(err) {
		panic("TIMEOUT")
	}
```

Reply:
yeah i see the problem.. this kind of a bit of complicated for this simple case. I deleted the timeout case for now, (meanwhile i will test this out to see if there are any simpler solutions)

Reply:
should we remove the `testTimeout` constant in the mean time?